### PR TITLE
Update the cassandra jar with the latest code.

### PR DESCRIPTION
**Pull request by brendandburns**
https://github.com/kubernetes/kubernetes/issues/18543

@zmerlynn

### CODE REVIEW

File: examples/cassandra/cassandra-controller.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-          image: gcr.io/google_containers/cassandra:v6
+          image: gcr.io/google_containers/cassandra:v7
```
Comment (wojtek-t) [inline examples/cassandra/cassandra-controller.yaml]: Do we want to commit the jar file too?
examples/cassandra/image/kubernetes-cassandra.jar

To be honest, I'm not sure that's the best idea...

Comment (zmerlynn) [inline examples/cassandra/cassandra-controller.yaml]: The current `Dockerfile` copies `kubernetes-cassandra.jar` in, so I imagine @brendandburns updated this by rebuilding *that*, but I don't know what process he used for that, which sort of makes the whole Docker build process kind of irrelevant. :)

Comment (brendandburns) [inline examples/cassandra/cassandra-controller.yaml]: Please see the TODO in the Makefile :P

(and its build using `mvn package` in the `.../java/` directory)

Comment (wojtek-t) [inline examples/cassandra/cassandra-controller.yaml]: yeah... SGTM

**Comment by wojtek-t** [comment]
LGTM

but you need to fix tests @brendandburns

**Comment by brendandburns** [comment]
docs updated, that should fix tests.

**Pull request merged**